src/network/networkd-dhcp6.c

```diff
@@ -682,6 +684,19 @@ int dhcp6_configure(Link *link) {
                         return log_link_error_errno(link, r, "DHCP6 CLIENT: Failed to set MUD URL: %m");
         }
 
+        SET_FOREACH(request_options, link->network->dhcp6_request_options, i) {
+                uint32_t option = PTR_TO_UINT32(request_options);
+
+                r = sd_dhcp6_client_set_request_option(client, option);
+                if (r == -EEXIST) {
+                        log_link_debug(link, "DHCP6 CLIENT: Failed to set request flag for '%u' already exists, ignoring.", option);
```

Comment:
Not that it matters, but can this even happen? It'd mean that a duplicate value in the set was encountered...

Reply:
By default there are requests  for example DNS, Domain  etc are set . Those can be by added here by the user . Hence this is a check here.